src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+item.natura.seed_bags.wheat_seed_bag.tooltip=Pflanzt in einem 3x3 Gebiet
+item.natura.seed_bags.potatoes_seed_bag.name=Kartoffelsack
+item.natura.seed_bags.potatoes_seed_bag.tooltip=Pflanzt in einem 3x3 Gebiet
+item.natura.seed_bags.carrots_seed_bag.name=Karottenbeutel
+item.natura.seed_bags.carrots_seed_bag.tooltip=Pflanzt in einem 3x3 Gebiet
+item.natura.seed_bags.nether_wart_seed_bag.name=Netherwarzenbeutel
+item.natura.seed_bags.nether_wart_seed_bag.tooltip=Pflanzt in einem 3x3 Gebiet
+item.natura.bonemeal_bag.name=Knochenmehlbeutel
+item.natura.bonemeal_bag.tooltip=Lässt Pflanzen in einem 3x3 Gebiet wachsen
+item.natura.overworld_seed_bags.barley_seed_bag.name=Gerstensamenbeutel
+item.natura.overworld_seed_bags.barley_seed_bag.tooltip=Pflanzt in einem 3x3 Gebiet
+item.natura.overworld_seed_bags.cotton_seed_bag.name=Baumwollsamenbeutel
+item.natura.overworld_seed_bags.cotton_seed_bag.tooltip=Pflanzt in einem 3x3 Gebiet
+
+item.natura.overworld_seeds.barley_seeds.name=Gerstensamen
+item.natura.overworld_seeds.barley_seeds.tooltip=Ähnlich wie Weizen wächst er in der Natur
+item.natura.overworld_seeds.cotton_seeds.name=Baumwollsamen
+item.natura.overworld_seeds.cotton_seeds.tooltip=Die Quelle aller Schnüre und Wolle
+
+item.natura.overworld_doors.eucalyptus_door.name=Eukalyptustür
+item.natura.overworld_doors.eucalyptus_door.tooltip=Das rosa Holz
+item.natura.overworld_doors.hopseed_door.name=Hopfensaattür
+item.natura.overworld_doors.hopseed_door.tooltip=Aufgestiegener Fehler
+item.natura.overworld_doors.sakura_door.name=Sakuraholztür
+item.natura.overworld_doors.sakura_door.tooltip=Blühende Kirsche
+item.natura.overworld_doors.redwood_door.name=Rotholztür
+item.natura.overworld_doors.redwood_door.tooltip=Riesenmammutbaum
+item.natura.overworld_doors.redwood_bark_door.name=Tür aus Rotholstamm
+item.natura.overworld_doors.redwood_bark_door.tooltip=Geheimer Tunnel
+item.natura.overworld_doors.tiger_door.name=Tigerholztür
+item.natura.overworld_doors.tiger_door.tooltip=Wildgetreide
+item.natura.overworld_doors.maple_door.name=Ahornholztür
+item.natura.overworld_doors.maple_door.tooltip=Etwas süßlich
+item.natura.overworld_doors.silverbell_door.name=Silberglockentür
+item.natura.overworld_doors.silverbell_door.tooltip=Silberne Glocken
+
+
+item.natura.nether_doors.ghostwood_door.name=Geisterholztür
+item.natura.nether_doors.ghostwood_door.tooltip=Bleich wie ein Gespenst
+item.natura.nether_doors.bloodwood_door.name=Blutholztür
+item.natura.nether_doors.bloodwood_door.tooltip=Feuerwiderstandsfähige Tür
+item.natura.nether_doors.darkwood_door.name=Dunkelholztür
+item.natura.nether_doors.darkwood_door.tooltip=Netherbaum
+item.natura.nether_doors.fusewood_door.name=Zündholztür
+item.natura.nether_doors.fusewood_door.tooltip=Netherbaum\nExplodierende Persönlichkeit
+
+item.natura.saguaro_fruit_item.name=Saguarofrucht
+
+################
+#    Tools     #
+################
+
+item.natura.flint_and_blaze.name=Feuerstein und Lohe
```

Comment:
Flint & Steel are translated as "Feuerzeug", how about calling this "Lohenfeuerzeug"?

Reply:
Yeah, to be honest I wasn't sure how t translate it properly, but you're right.

BTW. I had not really a clue what "Ascended Glitch" was refering to and just translated it to "Aufgestiegener Fehler". But I don't think that's the best solution, but couldn't think of a better one instantly. May you have an idea?

Reply:
Well it's a reference so it's really hard to translate. It's always valid to just pick something completely different as long as it is somewhat related to the thing.

Reply:
OK, thanks. :)